tools/state-viewer/src/commands.rs

```diff
@@ -1037,10 +1038,14 @@ mod tests {
         let home_dir = tmp_dir.path();
 
         let store = near_store::test_utils::create_test_store();
+        initialize_genesis_state(store.clone(), &genesis, Some(home_dir));
         let epoch_manager = EpochManager::new_arc_handle(store.clone(), &genesis.config);
-        let runtime =
-            NightshadeRuntime::test(tmp_dir.path(), store.clone(), &genesis, epoch_manager.clone())
-                as Arc<dyn RuntimeAdapter>;
+        let runtime = NightshadeRuntime::test(
```

Comment:
Looks like we frequently call `initialize_genesis_state` and `NightshadeRuntime::test`. I don't know what could be the generic setup, but at least uniting them into `initialize_genesis_and_runtime` could reduce number of lines.

Reply:
I actually would vote against doing that, just like how we have a big setup() for some of the integration tests. It's short, but it hides everything under the hood and then when some new test needs to do something different tomorrow, people will tend to add more arguments to the big setup() and it becomes super confusing how *each* test is set up exactly.

Spelling out the setup logic is tedious, but it makes each test very clear what specific setup that test has. And in my opinion, having test *setup* be easier to understand is critical for test maintenance.

Reply:
Makes sense!